optaplanner-benchmarks/optaplanner-perf-framework/src/main/java/org/jboss/qa/brms/performance/examples/conferencescheduling/generator/ConferenceSchedulingGenerator.java

```diff
@@ -213,12 +213,11 @@ public ConferenceSolution createConferenceSolution(int timeslotListSize, int roo
         labTalkCount = (dayListSize * labTimeslotCount) * labRoomCount;
 
         random = new Random(37);
-        ConferenceSolution solution = new ConferenceSolution();
-        solution.setId(0L);
+        ConferenceSolution solution = new ConferenceSolution(0l);
```

Comment:
`l` is often mistaken with `1`, hence it's better to use `L` for a `long` literal.
```suggestion
        ConferenceSolution solution = new ConferenceSolution(0L);
```